addons/csi/vsphere/vsphere-csi-driver.yaml

```diff
@@ -22,6 +22,9 @@
 #   - remove node selector for operator
 #   - set "improved-volume-topology": "false"
 #   - set image versions and features based on selected k8s version
+#   - set "block-volume-snapshot": "true"
```

Comment:
Is this really set? I don't see it, but maybe it's a default and block-volume-snapshot=false was removed.